spec/features/user_adds_a_release_spec.rb

```diff
@@ -15,9 +36,37 @@
     fill_in 'Password', with: user.password
 
     click_button 'Log in'
+
+    band = FactoryGirl.create(:band)
+
+    visit band_path(band)
+    click_link "Add Release"
+
+    fill_in "Title", with: Faker::Lorem.sentence(2)
+    fill_in "Track list", with: Faker::Lorem.sentence(2)
+    fill_in "Record label", with: Faker::Lorem.sentence(1)
+    fill_in "Record label url", with: Faker::Internet.url
+    fill_in "Catalog number", with: 50
+    choose("CD")
+    choose("LP")
+
+    click_button "Add Release"
+
+    expect(page).to have_content("You must be an admin")
+    expect(page).to have_content("Add a release")
+    expect(page).to have_content(50)
   end
 
-  scenario 'valid information in form to add a release' do
+  scenario 'admin inputs valid information' do
+    user = FactoryGirl.create(:admin)
+
+    visit new_user_session_path
+
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+
+    click_button 'Log in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.